dask_sql.Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG NUMPY_VER=1.21
+ARG RAPIDS_VER=21.08
+ARG UCX_PY_VER=0.21
+ARG JAVA_VER=11
+
+ADD https://raw.githubusercontent.com/dask-contrib/dask-sql/main/conda.txt /dask_sql_requirements.txt
+
+RUN conda config --set ssl_verify false
+
+RUN conda install -c gpuci gpuci-tools
+
+RUN gpuci_conda_retry install -c conda-forge mamba
+
+RUN gpuci_mamba_retry env create -n dask_sql --file /dask_sql_requirements.txt -c conda-forge
+
+RUN gpuci_mamba_retry install -y -n dask_sql -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
+    cudatoolkit=$CUDA_VER \
+    cudf=$RAPIDS_VER \
+    dask-cudf=$RAPIDS_VER \
+    numpy=$NUMPY_VER \
+    openjdk=$JAVA_VER \
+    # following requirements are for postgres testing; might not need them
+    sqlalchemy>=1.4.23 \
+    pyhive>=0.6.4 \
+    psycopg2>=2.9.1 \
```

Comment:
Do we anticipate ever needing to do GPU testing of PostgreSQL stuff? If not, then these packages (along with the `pip install` and `docker pulls`) can be removed. If so, then it would probably be a lot cleaner to add these requirements to a conda YAML file in dask-sql and install them all in the above environment creation.

Reply:
My guess would be that we don't need this. My only worry about moving this downstream to `dask-sql` is someone else could break our CI by changing the environment file there. But I think it's probably fine.

Reply:
Okay, in that case for now I'll remove these extra installs; if the need comes up later on it shouldn't be too difficult to add them in again